pki.spec

```diff
@@ -1305,6 +1305,7 @@ fi
 ################################################################################
 
 %{_datadir}/pki/tests/
+%{_bindir}/pki-lint
```

Comment:
Since this is a test tool it might be better to store it under `/usr/share/pki/tests/bin` or `/usr/share/pki/tests/tools`. This tool has a dependency on rpm tool, so it's not a normal PKI user tool that should be in `/usr/bin`.